cmd/cost/cost.go

```diff
@@ -0,0 +1,95 @@
+package cost
+
+import (
+	"fmt"
+	"github.com/openshift/osd-utils-cli/cmd/common"
+	awsprovider "github.com/openshift/osd-utils-cli/pkg/provider/aws"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+	"log"
+)
+
+// costCmd represents the cost command
+func NewCmdCost(streams genericclioptions.IOStreams) *cobra.Command {
+	opsCost = newCostOptions(streams)
+	costCmd := &cobra.Command{
+		Use:   "cost",
+		Short: "Cost Management related utilities",
+		Long: `The cost command allows for cost management on the AWS platform (other 
+platforms may be added in the future)`,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
```

Comment:
You can add `cmdutil.CheckErr(opsCost.complete(cmd, args))` here since opsCost is a global variable.

Reply:
Thanks for all the suggestions, Ben! I’ve made changes accordingly. Also good eye for noticing the hard-coded OU. I’m gonna ask Aaron to see if it’s okay to have that